docs/architecture.md

```diff
@@ -30,8 +30,7 @@ Each supported theme (Alaska, Hawaiian, etc.) has its own primitives folder (`sr
 
 These theme-specific primitives work alongside the shared `base` primitives to create a complete primitive value set for each brand.
 
-
-### 3. Basic Tokens
+### 2. Basic Tokens
```

Comment:
Just my 2 cents, so take it or leave it,...

I don't think we should be numbering these header sections. Numbers imply steps that should be followed in order. These headings are just used to group topics. By numbering it creates a little maintenance that is easy to make a mistake on in the future as well when adding/removing sections.